package.json

```diff
@@ -15,7 +15,7 @@
   "engines": {
     "node": "v20"
```

Comment:
With this change, will Node.js >20 also be supported? IIRC the only issue was the `with` / `assert`?

Reply:
Vercel just allow v20 for moment so I think we should stay on 20 for the moment.

Reply:
nvmrc can be v22 for local dev env tho

Reply:
But maybe keeping all v20 since CI should also be v20 to ensure vercel is also happy.